subprojects/packagefiles/icu/meson.build

```diff
@@ -34,6 +34,20 @@ if get_option('default_library') == 'static'
   add_project_arguments('-DU_STATIC_IMPLEMENTATION', language: 'c')
   add_project_arguments('-DU_STATIC_IMPLEMENTATION', language: 'cpp')
   usage_args = ['-DU_STATIC_IMPLEMENTATION']
+  if host_machine.system() == 'windows' and cpp.get_argument_syntax() == 'msvc'
```

Comment:
is it even possible for argument syntax to be msvc on non windows?

Reply:
No, I'll fix it.